.github/workflows/build-manywheel-images.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        rocm_version: ["4.5.2", "5.0"]
+        rocm_version: ["4.5.2", "5.0", "5.1"]
```

Comment:
@KyleCZH Please remove ROCm4.5.2 from this list. We will keep only the last two ROCm versions in this list at any point of time.